packages/engine-server/src/engineClient.ts

```diff
@@ -167,7 +167,8 @@ export class DendronEngineClient implements DEngineClient, EngineEventEmitter {
       };
     }
     if (!resp.data) {
-      throw new DendronError({ message: "no data" });
+      return {} as DEngineInitResp;
+      // throw new DendronError({ message: "no data" });
```

Comment:
Did you mean to remove the throw error line?

Also this is more for down the line, but if the init resp will look different based on whether we use sqllite or not, we should probably change this return type to make it more clear

Reply:
Forward looking, we shouldn't return a huge note payload on init (we don't need it anymore since the cache is gone on the `engineClient` side.  

But it's a bit tricky to make things compatible in this transitionary period between v2->v3. The SQLite init doesn't return anything on purpose, which is why I adjusted the logic here.  I need to do some additional scrutiny to make sure things work.